src/libYARP_wire_rep_utils/WireTwiddler.h

```diff
@@ -167,6 +167,7 @@ class YARP_wire_rep_utils_API WireTwiddlerReader : public yarp::os::InputStream
         pending_string_length = 0;
         pending_string_data = 0;
         override_length = -1;
+        lengthBuffer = yarp::os::NetInt32();
```

Comment:
Are you aware this copy instruction still leaves `lengthBuffer` not initialized? See https://github.com/robotology/yarp/blob/devel/src/libYARP_OS/src/NetInt32.cpp#L28 .

Reply:
Actually this happens only on *some* architectures